src/server/interfaces/fu.ts

```diff
@@ -0,0 +1,18 @@
+export interface fu {
```

Comment:
`fa` and `fu` are a bit ambiguous. What do they mean linguistically?

Reply:
place holders, will change. Slipped through